### DISCUSSION TITLE
Roblox Bloxstrap client

### DISCUSSION THREAD

**Discussion by ghost** [catppuccin/catppuccin#2035] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

Can someone make a port for the roblox bloxstrap client

### Any additional comments?

If you dont know what bloxStrap is here is the github for it https://github.com/pizzaboxer/bloxstrap

And pls Change the in game menu setting to version 3
![image](https://user-images.githubusercontent.com/114904150/235585517-a6ebe88d-28b5-4f23-91bb-95176ba36e88.png)

Here is What i want changed First all the menu in the discover tab, ect
![image](https://user-images.githubusercontent.com/114904150/235585836-c1855c97-4424-4005-9cec-263e34e3cc29.png)
![image](https://user-images.githubusercontent.com/114904150/235585901-eef7c0be-4d32-4b96-91d7-44bec6307a9f.png)

And i mean everything next The in game menu this is not the orinal menu btw sorry for spelling im lazy
![image](https://user-images.githubusercontent.com/114904150/235586556-0baa725d-01ca-4907-9619-af577c37d340.png)

One last thing is to make a custom theme you got to make a mod i dont know how to do that but just check the bloxstrap github wiki to see more

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by bloxsis** [comment]
Would love to see a port for [Blox strap](https://bloxstrapper.com/)! Here's the GitHub if anyone needs it: https://github.com/pizzaboxer/bloxstrap — and yeah, switching to version 3 menu would be great too.

**Comment by lillysss22** [comment]
I’d really love to see a port of [Bloxstrap](https://bloxstraproblox.com/) for other platforms or use cases! For anyone interested, here’s the official GitHub repo: https://github.com/pizzaboxer/bloxstrap. 
Also, switching over to the version 3 menu would be a great improvement, it's cleaner and more user-friendly compared to the older UI.
